site/components/link/test.js

```diff
@@ -3,8 +3,18 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import Link from '.';
 
+const mockContext = {
+  stateNavigator: {
+    stateContext: {
+      state: {},
+    },
+  },
+};
+
 describe('components/link', () => {
   it('renders OK children', () => {
-    shallow(<Link to="foo">Hello</Link>);
+    shallow(<Link to="foo">Hello</Link>, {
+      context: mockContext,
+    });
```

Comment:
Could we have a test that covers the `hasActiveChild` functionality?

Reply:
Sure.

Reply:
You are also missing an expectation in 'renders OK children'